common/test/Configuration.Oms.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public static partial class Configuration
+    {
+        public static class Oms
+        {
+            public static string AadTenant => GetValue("OMS_AAD_TENANT");
```

Comment:
All need to be clearly explained: how to configure on the service, what to copy/paste, etc.

Reply:
Please see prerequisites/readme.md